boot/bootdata/hivesft.inf

```diff
@@ -469,6 +469,7 @@ HKCU,"SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\BitBucket\Volume",,0x00
 
 ; CMD Settings
 HKLM,"SOFTWARE\Microsoft\Command Processor","AutoRun",0x00020000,""
+HKLM,"SOFTWARE\Microsoft\Command Processor","EnableExtensions",0x00010003,1
```

Comment:
Please don't, yet, because this will currently have side-effects on CMD.EXE that I haven't yet fully resolved.

Reply:
Reverted.